tutorials/create-first-ml-experiment/tutorial-1st-experiment-sdk-train.ipynb

```diff
@@ -57,8 +57,11 @@
       "source": [
         "Import the `Workspace` class, and load your subscription information from the file `config.json` using the function `from_config().` This looks for the JSON file in the current directory by default, but you can also specify a path parameter to point to the file using `from_config(path=\"your/file/path\")`. If you are running this notebook in a cloud notebook server in your workspace, the file is automatically in the root directory.\n",
         "\n",
-        "If the following code asks for additional authentication, simply paste the link in a browser and enter the authentication token."
-      ]
+        "If the following code asks for additional authentication, simply paste the link in a browser and enter the authentication token. In addition, if you have more than one tenant linked to your user, you will need to add the following lines:\n",
+        "from azureml.core.authentication import InteractiveLoginAuthentication\n",
+        "interactive_auth = InteractiveLoginAuthentication(tenant_id="your-tenant-id")\n",
```

Comment:
you need to escape the quotes in this line:
        "interactive_auth = InteractiveLoginAuthentication(tenant_id=\"your-tenant-id\")\n",